src/main/java/bowling/domain/state/Miss.java

```diff
@@ -0,0 +1,41 @@
+package bowling.domain.state;
+
+import bowling.domain.pin.Pin;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Miss extends EndState {
+    private final Pin firstDownPins;
+    private final Pin secondDownPins;
+
+    private Miss(final Pin firstDownPins, final Pin secondDownPins) {
+        verify(firstDownPins, secondDownPins);
+
+        this.firstDownPins = firstDownPins;
+        this.secondDownPins = secondDownPins;
+    }
+
+    private void verify(final Pin firstDownPins, final Pin secondDownPins) {
+        boolean isAllDown = firstDownPins.add(secondDownPins)
+                .isAllDown();
+
+        if (isAllDown) {
+            throw new IllegalArgumentException("Miss state should contain not down pins");
```

Comment:
```suggestion
            throw new IllegalArgumentException("Miss state should contain not all down pins");
```
Miss state는 모든 핀이 쓰러지지 않았어야 한다.라는 의미가 맞겠...죠???

Reply:
모두가 아니라 하나라도 쓰러지지 않은 핀이 있어야한다는 의미로 써서 그냥 down pins가 맞지 않ㅇ르까요? 영알못이라 ㅠ

Reply:
으..이건 영어니까..넘어가시죠 ㅋㅋ